CHANGELOG.md

```diff
@@ -2,6 +2,29 @@
 
 Describes notable changes.
 
+#### 1.16.0 - 2020/10/28
+Use separate DAOs for Core/Test/Management.
+- ITaskDao - data access operations used by the core and extensions.
+- IManagementTaskDao - data access operations used by the management extension.
+- ITestTaskDao - data access operations used for testing purposes
+
+Users of `tw-tasks-core-test` need to configure `ITestTaskDao` in the test configuration
```

Comment:
Seems like we may provide tw-tasks-core-test-spring-boot-starter in the future :)